portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionOSGI.testcase

```diff
@@ -0,0 +1,140 @@
+
+		task ("Given config file for client extension of type custom element created") {
+			ClientExtension.createGlobalVariableClientExtensionType(type = "customElement");
+
+			ClientExtension.checkConfigFileCreatedProperly();
+
+			RemoteApps.goToRemoteAppsPortlet();
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "LPS-169554 Access the osgi deployed custom element and assert that field information is displayed correctly and not editable"
+	@priority = "4"
+	test FieldsAreNotEditableWhenAccessed {
+		property custom.element.client.extension.created = "true";
+		property test.name.skip.portal.instance = "ClientExtensionOSGI#FieldsAreNotEditableWhenAccessed";
+
+		task ("When access the osgi deployed custom element") {
+			Click(
+				key_tableEntryName = "Vanilla Counter",
+				locator1 = "RemoteApps#TABLE_ENTRY_NAME_REMOTE_TABLE");
+		}
+
+		task ("Then field information is displayed correctly") {
+			RemoteApps.assertCustomElementFields(
+				entryCategoryName = "category.remote-apps",
+				entryCssurlText = "https://liferay.github.io/liferay-frontend-projects/index.css",
+				entryDescription = "Custom Element Description",
+				entryFriendlyElementName = "vanilla",
+				entryHtmlName = "vanilla-counter",
+				entryName = "Vanilla Counter",
+				entryProperties = "test-data-user=QAuser",
+				entrySourceCodeURL = "",
+				entryType = "Custom Element",
+				entryURLText = "https://liferay.github.io/liferay-frontend-projects/vanilla-counter/index.js");
+
+			FormFields.viewCheckboxChecked(fieldName = "Instanceable");
+
+			FormFields.viewCheckboxChecked(fieldName = "Use ES Modules");
+		}
+
+		task ("And Then fields are not editable") {
+			RemoteApps.ViewDisabledCustomElementFields(
+				entryCategoryName = "category.remote-apps",
+				entryCssurlText = "https://liferay.github.io/liferay-frontend-projects/index.css",
+				entryDescription = "Custom Element Description",
+				entryFriendlyElementName = "vanilla",
+				entryHtmlName = "vanilla-counter",
+				entryName = "Vanilla Counter",
+				entryProperties = "test-data-user=QAuser",
+				entrySourceCodeURL = "",
+				entryType = "Custom Element",
+				entryURLText = "https://liferay.github.io/liferay-frontend-projects/vanilla-counter/index.js");
+		}
+	}
+
+	@description = "LPS-169551: Verify that view button is visible"
+	@priority = "3"
+	test HasViewAction {
+		property custom.element.client.extension.created = "true";
+		property test.name.skip.portal.instance = "ClientExtensionOSGI#HasViewAction";
+
+		task ("Then View action is an available option") {
+			AssertElementPresent(
+				key_text = "View",
+				locator1 = "Button#ANY");
+		}
+
+		task ("And Then Edit action is not an available option field information is displayed correctly") {
+			AssertElementNotPresent(
+				key_text = "Actions",
+				locator1 = "Button#ANY");
```

Comment:
is the key_text for Edit action, `Actions`? I would've thought it would be more specific like `Edit`

Reply:
oh nvm, I understand better, you're checking to see if ellipsis is available. Is this the right key_text?

Regardless, I think you need to refer to the term `ellipsis` somewhere is better understood what we're trying to check. ie: FrontendDataSet#ENTRY_VERTICAL_ELLIPSIS is more clear to me what I'm trying to interact with.

Reply:
Done!